islandora_binary_object.module

```diff
@@ -96,7 +96,9 @@ function islandora_binary_object_xml_form_builder_forms() {
 /**
  * Implements hook_islandora_ingest_steps().
  */
+ // @codingStandardsIgnoreStart
 function islandora_binary_object_islandora_binaryObjectCModel_islandora_ingest_steps() {
```

Comment:
These should probably just be lower-case. From [PHP docs](http://php.net/manual/en/functions.user-defined.php):

> Function names are case-insensitive [...]